index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Mobile-First</title>
+
+  <link rel="stylesheet" href="styles/general.css">
+  <link rel="stylesheet" href="styles/nav.css">
+  <link rel="stylesheet" href="styles/info.css">
+  <link rel="stylesheet" href="styles/footer.css">
+</head>
+<body>
+  <main>
+    <header id="header">
+      <div class="header-container">
+        <a href="https://twitter.com/black_okyere"><p>Skabeo</p></a>
+          <nav class="hamburger">
+            <span></span>
+            <span></span>
+            <span></span>
+          </nav>
+      </div> 
+    </header>
+  
+    <section id="bio" class="bio">
+      <div class="bio-container">
+        <h1 class="author"><span class="text-gradient">
+          Spencer Okyere</span></h1>
+        <h2 class="occupation">Software Developer</h2>
+        <div class="gradient" />
+      </div>
+    </section>
+  
+    <section id="description">
+      <div class="description-container">
+        <p>I can help you build a product , feature or 
+          website Look through some of my work and 
+          experience! If you like what you see and have 
+          a project you need coded,  don&#39;t hestiate to 
+          contact me.</p>
+        </div>
+  
+      <hr class="line">
+    </section>
+  
+    <section id="footer" class="footer">
+      <div class="footer-container">
+        <div class="items">
+          <h4 class="text-gradient">Language</h4>
+          <ul>
+            <li>Javascript</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </ul>
+        </div>
+        <div class="items">
+          <h4 class="text-gradient">Frameworks</h4>
+          <ul>
+            <li>Bootstrap</li>
+            <li>Ruby on Rails</li>
+            <li>RSPec</li>
+            <li>CapyBara</li>
+            <li>Solenium</li>
+          </ul>
+       </div>
+        <div class="items">
+          <h4 class="text-gradient">Skills</h4>
+          <ul>
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Terminal</li>
+          </ul>
+        </div>
+      </div>
+    </section>
+
+    <hr class="line">
+
+    <section id="social-links">
+      <div class="social-links">
+        <ul class="img-container">
+          <li><a href="https://www.instagram.com/___spec/"><img src="img/instagram.svg" ></a></li>
+          <li><a href="https://twitter.com/black_okyere"><img src="img/twitter.svg" ></a></li>
+          <li><a 
+            href="https://www.linkedin.com/in/okyere-spencer-9b602623b/">
+            <img src="img/linkedin.svg" ></a></li>
```

Comment:
- [ ] Kindly add all social media apps as mentioned in the Figma design:

![Screenshot from 2023-02-16 23-57-49](https://user-images.githubusercontent.com/45226848/219461718-d841d329-84a5-4687-8c6d-d5a060988756.png)